core-bundle/src/Resources/contao/classes/FrontendUser.php

```diff
@@ -237,7 +237,7 @@ protected function setUserFromDb()
 		}
 
 		// Get the group login page
-		if ($this->groups[0] > 0)
+		if (\count($this->groups) && $this->groups[0] > 0)
```

Comment:
Is `$this->groups` guaranteed to be indexed starting with zero? Otherwise I’d just use:
```suggestion
		if (($this->groups[0] ?? 0) > 0)
```

Reply:
I've never seen any different - but who knows what's out there?

Reply:
Adjusted in b367929334.